README.md

```diff
@@ -49,6 +49,33 @@ This list is just for Red Hat contributions to projects hosted on GitHub.  See h
 ### Hosting projects on /RedHatOfficial
 
 Open an issue in *this* repo, and the owners of the RedHatOfficial organization will get in touch with you.
+
+## Quick start
+    ```
+    git clone https://github.com/RedHatOfficial/RedHatOfficial.github.io.git
+    cd RedHatOfficial.github.io
+    
+    ```
+ 
+## Building and Running Locally
```

Comment:
```suggestion
## Building and running locally
```

Brand team prefers sentence case for headlines and I try to be consistent in documentation when possible.